tools/circleci_dependencies.sh

```diff
@@ -31,7 +31,7 @@ elif [[ "$CIRCLE_JOB" == "linkcheck"* ]]; then
 else  # standard doc build
 	echo "Installing doc build dependencies"
 	python -m pip uninstall -y pydata-sphinx-theme
-	python -m pip install --upgrade --progress-bar off --only-binary matplotlib -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
+	python -m pip install --upgrade --progress-bar off --only-binary matplotlib mne-qt-browser -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
```

Comment:
for example you can replace `mne-qt-browser` here with `https://github.com/marsipu/mne-qt-browser/zipball/whateverbranchname` and it should use your threading branch (if you set the name properly)

Reply:
```suggestion
	python -m pip install --upgrade --progress-bar off --only-binary matplotlib https://github.com/marsipu/mne-qt-browser/zipball/fix_threading -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt
```